en/components/general/wpf_to_angular_guide/create_ui_with_components.md

```diff
@@ -0,0 +1,78 @@
+
+```cmd
+ng g c sample
+```
+
+When the command completes, you'll notice a number of things have happened. First, we have a new folder with the name we gave to our component. We also have four new files - an html file, a SPEC file, a typescript file, and a CSS file. We also made an update to the `app.module.ts` file.
+
+<img src="../../../images/general/sample_component.png" />
+
+Now, let's look at the `sample` folder which contains our newly created component. We can see that all three files that are required for a component are there plus this extra SPEC file. This is actually a test file which we do not need for this article, so we're not going to bother with it now. We also made a modification to the app.module.ts, in which it added the sample component to the `declarations` section of our NgModule. This is important - whenever a new component is created, it must be declared in the declarations section of the NG module. This section essentially says what components belong to this module.
+
+So let's go back to our `sample.component.ts` file. It looks very similar to the `app.component.ts` file we covered above. In this case we have our import statement where we're importing our component. We have our component decorator where we are defining our selector as `app-sample`. We have our template URL as `sample.component.html` and we have our style URL  `sample.component.css`.
+
+Let's go ahead and open up the html file where we can see a paragraph stating 'sample works!'. We want to see this when we start the application so let's go to the app.component.html. We will delete most of the initial markup except for the title, and just add `<app-sample></app-sample>`.
+```html
+<div style="text-align:center">
+  <h1>
+    Welcome to {{ title }}!
+  </h1>
+  <app-sample></app-sample>
+</div>
+```
+If we now save it and type `npm start` in the terminal, it will compile and build our application and launch it inside the browser.
+If all went well, our application should be running in the browser. We have `welcome to demo-app` - the markup from the app component and then `sample works!`. This is the markup from our sample component that we just added.
+Let's now change our sample component a little by changing the text to 'This sample works very well!'.
+```html
+<p>
+  This sample works very well!
+</p>
+```
+Once we save and check the browser, we will see that we are indeed editing the html that is responsible for rendering the sample component.
+If we wanted to style our component, we would come look at our style URL. So let's open up our sample.component.css file and add something to a paragraph. We'll set the color to say red.
+```css
+p {
+    color: red;
+}
+```
+Once we save that and open up the browser, we should see that the styling in the CSS file has been applied to our sample component.
+
+## Additional Resources
```

Comment:
Update the **Additional Resources** section with more relevant links.

Reply:
Addressed.